plugin/storage/es/spanstore/dbmodel/from_domain_test.go

```diff
@@ -108,6 +108,26 @@ func TestTagMap(t *testing.T) {
 	assert.Equal(t, tagsMap, dbSpan.Process.Tag)
 }
 
+func TestNilProcess(t *testing.T) {
+	tags := []model.KeyValue{
+		model.String("foo", "foo"),
+		model.Bool("a", true),
+		model.Int64("b.b", 1),
+	}
+
+	spanWithNilTags := model.Span{Tags: tags, Process: &model.Process{Tags: nil}}
```

Comment:
This test would pass without any code changes in the PR. It's the nil Process that's causing panic in production, not the nil tags (nil tags are still iterable).